internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1280,6 +1280,16 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Optional: true,
 				Default:  false,
 			},
+
+			"custom_ca_trust_certificates": {
```

Comment:
Can we append `_base64` to the name so it's clear what format we're expecting here
```suggestion
			"custom_ca_trust_certificates_base64": {
```